src/features/components/atoms/card/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { ReactNode } from 'react'
+import CardBasic from './card-basic'
+import CardPrimary from './card-primary'
+
+type CardTypes = {
+    Basic: typeof CardBasic
+    Primary: typeof CardPrimary
+}
+
+const Card: CardTypes = ({ children }: { children: ReactNode }) => {
+    return <>{children}</>
+}
+
+Card.Basic = CardBasic
```

Comment:
Can we rename folder and component name on `CardBase` and import it as `Base`, just to follow existing approach 🙏